SS14.Launcher/Models/Data/Migrations/Script0006_Hubs.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE Hubs (
```

Comment:
There's some fancy relational algebra reason for this, but table names shouldn't be plural (which the rest of the launcher tables also follow).